AnnoyancesFilter/Popups/sections/subscriptions_specific.txt

```diff
@@ -51,6 +51,8 @@ gaugau.futabanet.jp,newscientist.com,jiji.com,edp24.co.uk,technologyreview.com,n
 !
 ! SECTION: Subscriptions - Regular rules
 !
+yakutia.info##.m_share
+yakutia.info##.share
```

Comment:
Could you please add domain to a common rule:
https://github.com/AdguardTeam/AdguardFilters/blob/ecfdfe7bd18c9da38402a08280b15d12015b4902/SocialFilter/sections/specific.txt#L12-L13